src/components/link/link.stories.mdx

```diff
@@ -1,146 +1,146 @@
-import { useState } from 'react';
-import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-import Link, {InternalLink} from './link.component';
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import Link, { InternalLink } from "./link.component";
 
-<Meta title="Link" parameters={{ info: { disable: true }}} />
+<Meta title="Link" parameters={{ info: { disable: true } }} />
 
 # Link
+
 Navigates the user to another location.
 
-Avoid using links for commands (performing an action like saving a form) - use them for navigating the user to a new location. 
+Avoid using links for commands (performing an action like saving a form) - use them for navigating the user to a new location.
```

Comment:
Should we link to the routing documentation here? We don't want people that are using react-router to accidentally use `href` because it'll cause a full page refresh.